docker-compose.yml

```diff
@@ -0,0 +1,9 @@
+version: '3'
+services:
+    start-dev:
+        image: quay.io/keycloak/keycloak:latest
+        ports:
+            - '${KEYCLOAK_PORT}:8080'
+        environment:
+            - KEYCLOAK_ADMIN=${KEYCLOAK_ADMIN}
+            - KEYCLOAK_ADMIN_PASSWORD=${KEYCLOAK_ADMIN_PASSWORD}
```

Comment:
This docker compose file ends up in an stopped container. Adding this fixed it for me:
`command: start-dev` at the same level of `image`, `ports` and `environment`

Reply:
added `command: start-dev` and pinned keycloak version to `22.0.5`